sw/airborne/modules/loggers/file_logger.c

```diff
@@ -32,7 +32,13 @@
 #include "std.h"
 
 #include "subsystems/imu.h"
+#ifdef COMMAND_THRUST
```

Comment:
We already had some issues in the past with a similar modification that was reverted in the end. Can you check with @fvantienen @EwoudSmeur or @kirkscheper that this is not breaking something else ?
I think this logger should be rewritten to be more generic and don't hard-code the variables.

Reply:
I know. @EwoudSmeur is no longer in the lab, @fvantienen now available fultime again after his graduation. I'll will sit down with Kirk asap to discuss and improve some PPRZ things. Burden if not working ultimatley would be on me since need to support all Cyberzoo etc. Studends. So yes surely.

Reply:
I think issues where more with hybrid stuff I think, but not sure